README.md

```diff
@@ -45,6 +45,9 @@ Missing a feature? Raise a github issue [here](https://github.com/ChaitanyaKaran
     + [Response](#response-7)
   * [8. ServiceNow.setNetworkOptions](#8-servicenowsetnetworkoptions)
     + [Request](#request-8)
+  * [9. ServiceNow.setNetworkOptions](#9-servicenowgettablestats)
```

Comment:
@shahrukhx01 Can you change the name of the function. This is conflicting with the previous one.